app/src/main/java/ch/epfl/sdp/map/PathsHandler.java

```diff
@@ -57,126 +63,192 @@
  * as well as points of met infected users.
  */
 public class PathsHandler extends Fragment {
-    static final String POINTS_SOURCE_ID = "points-source";
-    static final String POINTS_LAYER_ID = "pointslayer";
-    static final String PATH_SOURCE_ID = "line-source";
-    private static final int ZOOM = 7;
-    static final String PATH_LAYER_ID = "linelayer"; // public for testing
-    public List<Point> pathCoordinates;
-    public List<Point> infected_met;
+    private static final String YESTERDAY_POINTS_SOURCE_ID = "points-source-one";
+    private static final String BEFORE_POINTS_SOURCE_ID = "points-source-two";
+    static final String YESTERDAY_POINTS_LAYER_ID = "pointslayer-one";
+    static final String BEFORE_POINTS_LAYER_ID = "pointslayer-two";
+
+    private static final String YESTERDAY_PATH_SOURCE_ID = "line-source-one";
+    private static final String BEFORE_PATH_SOURCE_ID = "line-source-two";
+    static final String YESTERDAY_PATH_LAYER_ID = "linelayer-one";
+    static final String BEFORE_PATH_LAYER_ID = "linelayer-two";
+
+    private List<Point> yesterdayPathCoordinates;
+    private List<Point> beforeYesterdayPathCoordinates;
+    private List<Point> yesterdayInfectedMet;
+    private List<Point> beforeYesterdayInfectedMet;
+
+    private double latitudeYesterday;
+    private double latitudeBefore;
+    private double longitudeYesterday;
+    private double longitudeBefore;
+
+    private boolean pathLocationSet1 = false;
+    private boolean pathLocationSet2 = false;
+
+    public String yesterdayString;
+    public String beforeYesterdayString;
+
+    private static final int ZOOM = 13;
     private MapboxMap map;
-    private FirestoreInteractor fsi = new ConcreteFirestoreInteractor();
     private MapFragment parentClass;
-    private double latitude;
-    private double longitude;
 
 
     PathsHandler(@NonNull MapFragment parentClass, @NonNull MapboxMap map) {
         this.parentClass = parentClass;
         this.map = map;
+        setCalendar();
         initFirestorePathRetrieval().thenAccept(this::getPathCoordinates);
     }
 
-    @VisibleForTesting
-    public static String getPathLayerId() {
-        return PATH_LAYER_ID;
+    private void setCalendar() {
+        Date rightNow = new Date(System.currentTimeMillis());
+        Calendar cal = Calendar.getInstance();
+        cal.setTime(rightNow);
+        cal.add(Calendar.DAY_OF_MONTH, -1);
+        Date yes = cal.getTime();
+        cal.add(Calendar.DAY_OF_MONTH, -1);
+        Date bef = cal.getTime();
+
+        yesterdayString = "2020/05/13"; //this is for demo only, should be replaced by: dateToSimpleString(yes);
```

Comment:
Can't we add somewhere a static variable for demo-ing ? 
Oh and actually Solal said there should not be demo - stuff on the master, but only on a separate demo branch

Reply:
Oh right we should make a new branch next time we want to do a special demo.
Otherwise this value isn't useful except for the demo, so not sure whether it deserves to in a static var (I might be wrong though)

Reply:
Yes you're right
Maybe the best is that you remind yourself that this has to be changed in the demo branch